systemtest/src/test/java/io/strimzi/systemtest/connect/ConnectBuilderST.java

```diff
@@ -471,6 +473,9 @@ void testBuildOtherPluginTypeWithAndWithoutFileName(ExtensionContext extensionCo
     @Tag(ACCEPTANCE)
     @ParallelTest
     void testBuildPluginUsingMavenCoordinatesArtifacts(ExtensionContext extensionContext) {
+        // using kind we encounter (error building image: deleting file system after stage 0: unlinkat //product_uuid: device or resource busy)
+        assumeFalse(KubeClusterResource.getInstance().isKind());
+
```

Comment:
Maybe create annotation and skip the whole suite? I guess it will help you also for `regression` pipelines

Reply:
> Maybe create annotation and skip the whole suite? I guess it will help you also for regression pipelines

I don't know if all KafkaConnectBuilder test cases will fail. I am not fully sure but on Thursday I execute whole suite and only 2-3 test cases failed. So I can for sure create that annotation but I will skip only those, which will fail.